src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(int length, int width) {
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
+        return new Rectangle(getRandomInt(), getRandomInt());
+    }
+
+    @Override
+    public void printInformation() {
+        System.out.println("Rectangle length: " + length + ", width: " + width
```

Comment:
All magic numbers in your code should be constants. The numbers used in the `printInformation` method for calculating the area should be replaced with named constants to improve code readability and maintainability.